kuma/static/js/wiki-edit.js

```diff
@@ -396,8 +396,8 @@
     // Injects a DIV with language to the effect of "you had a previous draft, want to restore it?"
     // This takes the place of an ugly, ugly confirmation box :(
     var $draftDiv;
-    function displayDraftBox(content) {
-        var text = gettext('You have a draft in progress. <a href="" class="restoreLink">Restore the draft content</a> or <a href="" class="discardLink">discard the draft</a>.');
+    function displayDraftBox(content, draft_time) {
+        var text = gettext('You have a draft in progress ('+ draft_time +'). <a href="" class="restoreLink">Restore the draft content</a> or <a href="" class="discardLink">discard the draft</a>.');
```

Comment:
All the existing users' drafts will show "(null)" here ...

![draft-null-datetime](https://cloud.githubusercontent.com/assets/71928/11792149/3cf37656-a26a-11e5-9e77-bcb53d8d6cda.png)

So maybe:
```
var draft_time_str = (draft_time ? '(' + draft_time + ')' : '';
```